### PR TITLE
Gdx pay android google billing subscriptions

### PR DESCRIPTION
**Pull request by keesvandieren**
Added basic support for subscriptions.

Validated with our app subscriptions are ok. 

Validated for another app, that in-app purchases are still being restored.

Can you have a look? 

Thanks in advance

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ task wrapper(type: Wrapper) {
 
 allprojects {
     group = 'com.badlogicgames.gdxpay'
-    version = '1.0.1-SNAPSHOT'
+    version = '1.1-SNAPSHOT'
```
Comment (MrStahlfelge) [inline build.gradle]: 1.1.0-SNAPSHOT

File: gdx-pay-android-googlebilling/src/com/badlogic/gdx/pay/android/googlebilling/PurchaseManagerGoogleBilling.java [hunk 2]
```diff
@@ -110,45 +110,74 @@ public void onBillingServiceDisconnected() {
...
-
-                            if (responseCode != BillingClient.BillingResponse.OK) {
-                                Gdx.app.error(TAG, "onSkuDetailsResponse failed, error code is " + responseCode);
-                                if (!installationComplete)
-                                    observer.handleInstallError(new FetchItemInformationException(
-                                            String.valueOf(responseCode)));
+        if (skuList.isEmpty()) {
+            Gdx.app.log(TAG, "No skus configured");
+            setInstalledAndNotifyObserver();
+        }
```
Comment (MrStahlfelge) [inline gdx-pay-android-googlebilling/src/com/badlogic/gdx/pay/android/googlebilling/PurchaseManagerGoogleBilling.java]: Why is this changed to call the sku detail query even if the sku list is empty?

Comment (MrStahlfelge) [reply gdx-pay-android-googlebilling/src/com/badlogic/gdx/pay/android/googlebilling/PurchaseManagerGoogleBilling.java]: Resolved in the meantime

**Comment by keesvandieren** [comment]
@MrStahlfelge  regarding version change (from 1.0.1-SNAPSHOT to 1.1-SNAPSHOT) I sent you a PM on gitter.

**Comment by MrStahlfelge** [comment]
Do you think it's time to release a new version with this change?

**Comment by keesvandieren** [comment]
It is in latest version: https://github.com/libgdx/gdx-pay/releases
